datamodel.sv-display-mgmt.xml

```diff
@@ -309,6 +311,20 @@
 				<entry id="Class:Model/Attribute:type/Value:Display" _delta="define">Monitor</entry>
 				<entry id="Class:Model/Attribute:type/Value:Display+" _delta="define"><![CDATA[]]></entry>
 			</entries>
+			<dictionary id="FR FR">
+				<entries>
+					<entry id="Class:Display" _delta="define">Ecran</entry>
+					<entry id="Class:Display/Attribute:size" _delta="define">Taille</entry>
+					<entry id="Class:Display/Attribute:size+" _delta="define">Taille de l'écran en pouces</entry>
+					<entry id="Class:Display/Attribute:type_id" _delta="define">Type</entry>
+					<entry id="Class:Display/Attribute:type_id+" _delta="define"><![CDATA[]]></entry>
```

Comment:
This workaround is not needed anymore as the bug is fixed in iTop 3.1.2, see https://github.com/Combodo/iTop/pull/600

```suggestion
					<entry id="Class:Display/Attribute:type_id+" _delta="define"/>
```